packages/python-yara-dex/PKGBUILD

```diff
@@ -0,0 +1,38 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
```

Comment:
You can find our PKGBUILD templates here : https://github.com/BlackArch/blackarch-pkgbuilds